docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +160,22 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-using-centosci]]
+=== Building the OpenShift Documentation using CentOS CI
+
+To get the view of {project} documentation integrated into the OpenShift documentation using CentOS CI, trigger the documentation build using following:
+
+----
+$ export API_KEY=<api-key>
+$ curl -H "$(curl --user minishift:$API_KEY 'https://ci.centos.org//crumbIssuer/api/xml?xpath=concat(//crumbRequestField,":",//crumb)')" -X POST https://ci.centos.org/job/minishift-docs/build --user "minishift:$API_KEY" --data-urlencode json='{"parameter": [{"name":"REPO", "value":"<user-repo>"}, {"name":"BRANCH", "value":"<docs-branch>"}]}'
```

Comment:
+1 :-)

What's about we export BRANCH and REPO as well and use it in the curl command. Then the curl command is cut&paste provided you have set the variables